releasenotes/notes/tighten-pip-tuf-index-71cc1530c1ebe930.yaml

```diff
@@ -0,0 +1,16 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+security:
+  - |
+    We replace the PyPI index with our own by default, in order to prevent
+    accidental installation of Datadog or even third-party packages from PyPI.
+  - |
+    Remove some undocumented power user options to the `datadog integration`
```

Comment:
format of the release note is reST: inline code needs to be quoted with double backticks instead of single backticks :)

also, it's the `datadog-agent integration` command